routes/routes.js

```diff
@@ -0,0 +1,67 @@
+const express = require('express');
+const { keysToCamel } = require('../common/utils');
+
+const router = express.Router();
+const {db} = require('../server/db');
+
+router.use(express.json());
+
+router.post('/', async (req, res) => {
+  try {
+    const data = req.body;
+    const newRoute = await db.query(
+      `INSERT INTO routes (${data.driverId ? 'driver_id, ' : ''} id) 
+      VALUES(${data.driverId ? `${data.driverId}, ` : ''} ${data.id}) RETURNING *;`,
+    );
+    res.status(200).json(keysToCamel(newRoute[0]));
```

Comment:
since `newRoute` could potentially be an empty array, `newRoute[0]` would evaluate to `undefined` which is unwanted behavior. instead use the `??` operator to return an empty array if `newRoute[0]` evaluates to `undefined` (check the programming practices pdf for more info`

Reply:
pls do the same with all other instances where you're returning the first index of an object